src/main/java/com/github/kongchen/swagger/docgen/reader/AbstractReader.java

```diff
@@ -393,6 +395,14 @@ protected void updateApiResponse(Operation operation, ApiResponses responseAnnot
             } else {
                 operation.response(apiResponse.code(), response);
             }
+            if (apiResponse.code() == 200) {
+                contains200 = true;
+            } else if (apiResponse.code() > 200 && apiResponse.code() < 300) {
+                contains2xx = true;
+            }
+        }
+        if (!contains200 && contains2xx) {
+            operation.getResponses().remove("200");
```

Comment:
operation.getResponses() == null?

Reply:
Technically should not be possible, because `contains2xx` is true, but better be safe than sorry - added a null check